configFile/fields/cicd/cicd.go

```diff
@@ -0,0 +1,6 @@
+package cicd
+
+type CICD struct {
```

Comment:
I think pointer is a bad idea. Invalid Pointer is more horrible than string's overhead.

Reply:
Feel free to ignore this comment.

Reply:
> I think pointer is a bad idea. Invalid Pointer is more horrible than string's overhead.

After all, these things will be handled in #33.